gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -298,6 +298,12 @@ gpii.settingsHandlers.writeFile = function (content, options) {
  */
 gpii.settingsHandlers.handleFileSolutionEntry = function (solutionEntry, modifier, parser, isWrite) {
     var options = solutionEntry.options;
+    var initialContent = options.initialContent;
+
+    var exists = fs.existsSync(options.filename);
+    if(!exists){
+        fs.writeFileSync(options.filename, initialContent, options.encoding || "utf-8");
+    }
 
```

Comment:
1) I think it should be ok since it runs the line of code first before doing all the transformation. From what I found out, the settings handler is synchronous. 

2) Haven't thought of that. I'll figure that out. Should just throw an error since settings handler will crash if file is not created. For the whole file, I don't think there's much exception handling.

There's also the case where if the initial file in the computer is empty or if it has the wrong content, the settings handler will crash.